src/plugins/node/loop/nodeloopplugin.cpp

```diff
@@ -136,30 +139,31 @@ extern "C" {
 	}
 
 	NODELOOPPLUGIN_DECL int ibmras_monitoring_plugin_init(const char* properties) {
-		return 0;
+       uv_check_init(uv_default_loop(), &check_handle);
```

Comment:
So, if anyone knows `libuv` better than I do (which honestly I don't know anything about how it works). There is probably a better way of releasing the event loops on start and stop. But this way seems to work ok.